conda_build/cli/main_build.py

```diff
@@ -85,7 +90,7 @@ def parse_args(args: Sequence[str] | None) -> tuple[ArgumentParser, Namespace]:
         "-t",
         "--test",
         action="store_true",
-        help="Test package (assumes package is already built).  RECIPE_DIR argument must be a "
+        help="Test package (assumes package is already built).  RECIPE_PATH argument must be a "
         "path to built package .tar.bz2 file.",
```

Comment:
```suggestion
        "path to built package file.",
```
Does it only support tarbz2